Makefile

```
@@ -111,7 +112,7 @@ test-e2e: ## Playwright で E2E テストを実行する（test には含めな
 	@# ここの画像を報告に添える（CLAUDE.md の「報告にスクリーンショットを添える」）。
 	cd $(WEB_DIR) && bun run test:e2e
 
-lint: lint-go lint-web ## golangci-lint とフロントエンドの lint / 整形検査を実行する
+lint: lint-go lint-web lint-docs lint-fmt lint-nix lint-actions ## Go / フロントエンド / Markdown / Nix / Actions を検査する
```

checkmake (0.3.2)

[warning] Target "lint" should be declared PHONY.

(phonydeclared)